modelgauge/main.py

```diff
@@ -228,6 +241,81 @@ def run_test(
     print("Full TestRecord json written to", output_file)
 
 
+@modelgauge_cli.command()
+@click.option(
+    "sut_names",
+    "-s",
+    "--sut",
+    help="Which registered SUT(s) to run.",
+    multiple=True,
+    required=True,
+)
+@click.option(
+    "--workers",
+    type=int,
+    default=None,
```

Comment:
This gives an error if no `--workers` arg is given because `thread_count` is inherited from workers. Maybe we should probably give this a different default value or make it a required argument.

```shell
poetry run modelgauge run-prompts id_draft_id_elec_2024-06-28.csv --sut gemma-7b-it --sut Mixtral-8x7B-Instruct-v0.1 --sut llama-3-70b-chat-hf
```
```python
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/localdisk/twilbers/src/repos/modelgauge/modelgauge/main.py", line 324, in main
    modelgauge_cli()
  File "/localdisk/twilbers/src/repos/modelgauge/.venv/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/localdisk/twilbers/src/repos/modelgauge/.venv/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/localdisk/twilbers/src/repos/modelgauge/.venv/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/localdisk/twilbers/src/repos/modelgauge/.venv/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/localdisk/twilbers/src/repos/modelgauge/.venv/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/localdisk/twilbers/src/repos/modelgauge/modelgauge/main.py", line 314, in run_prompts
    pipeline.run()
  File "/localdisk/twilbers/src/repos/modelgauge/modelgauge/pipeline.py", line 288, in run
    segment.start()
  File "/localdisk/twilbers/src/repos/modelgauge/modelgauge/pipeline.py", line 168, in start
    for i in range(self.thread_count):
```

Reply:
Ah, great catch. This used to work; I'll make it work again.